### DISCUSSION TITLE
Dogs representative idea: Boston Terrier

### DISCUSSION THREAD

**Discussion by pewinski** [gabrielchl/intellij-pets#5] [Ideas]
Hello there, awesome plugin :)
Are you planning expanding list of animals? Dogs are missing ^^
A Boston Terrier would be great, might fit in size perfectly to the pack ;)

![medium_boston-terrier](https://github.com/gabrielchl/intellij-pets/assets/3342202/247c5e67-bf05-4461-aa3d-4b7caab069d3)

**Comment by gabrielchl** [comment]
hey, just a quick update

good news: v0.1.4 has been released, which adds 5 variants of dogs along with some other pets (see pic below)
![collection](https://github.com/gabrielchl/intellij-pets/assets/5616556/80eb485a-5d89-489e-b115-d8df9283b57e)

bad news: none of them look like boston terriers! I'll try to find / make the sprite sheets for them when I have time!

**Reply by pewinski** [reply]
Lovely, thank you!
